tests/ocpp-server/README.md

```diff
@@ -1,23 +1,33 @@
 # OCPP2 Mock Server
 
 This project includes an Open Charge Point Protocol (OCPP) version 2.0.1 mock server implemented in Python.
+https://github.com/mobilityhouse/ocpp/blob/master/ocpp/v201/enums.py
 
 ## Prerequisites
 
 This project requires Python 3.7+ and [poetry](https://python-poetry.org/) to install the required packages:
+```shell
+pip install poetry
+```
+then
 
 ```shell
 poetry install
 ```
 
+
 ## Running the Server
 
 To start the server, run the `server.py` script:
 
 ```shell
 poetry run task server
 ```
+or
```

Comment:
If you want to document the OCPP2 command emission usage, please do a generic documentation with a specific section `Running the server with command sending`.